src/components/UnsupportedBrowserMsg.tsx

```diff
@@ -18,6 +18,25 @@ export const UnsupportedBrowserMsg: FC = () => {
   const history = useHistory();
   const [open, setOpen] = useState(true);
 
+  React.useEffect(() => {
+    navigator.registerProtocolHandler(
+      'web+safarita',
+      'https://pr-33.d3pqoo7mq9yhwf.amplifyapp.com/?src=%s',
+      'Safari'
+    );
```

Comment:
@ajsamra this is how to use `registerProtocolHandler`. See my links in main PR thread, but basically the params are:

1. The `scheme` to use. I think custom web-based ones have to be prefixed with `web+`
2. The `url` that will be used when an `href` is set to the `scheme`. I found mixed instructions but most say:
    - The URL must be same (or same domain at least) as the one it's being called from
    - ...and should be `https` (definitely mixed on this)
    - The `%s` is required. Whatever goes into the `STUFF` part of `<a href="web+safarita:STUFF">` will be applied to the `%`. Not sure it's useful to us but it _is_ required.
3. The app to open it in (???). I'm not sure if it's Google or Chrome, or if this is even the way to do it in the first place.